app/models/foreman_tasks/task/dynflow_task.rb

```diff
@@ -120,6 +157,13 @@ def get_humanized(method)
                                    end
     end
 
+    def find_humanize_method_kind(method)
+      return method if method.match?(/humanized_.*/)
```

Comment:
Tests failures seem related, At least line connects directly to this failure http://ci.theforeman.org/job/test_plugin_pull_request/4879/database=postgresql,label=fast,ruby=2.3/testReport/junit/(root)/ForemanTasks__DynflowConsoleAuthorizerTest__user%20without%20edit_foreman_tasks%20permissions/test_0001_can_not_see_any_tasks/

Most of the failures in http://ci.theforeman.org/job/test_plugin_pull_request/4879/ seemed to be related to that.